DataCache.podspec

```diff
@@ -6,6 +6,7 @@ Pod::Spec.new do |s|
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.source       = { :git => 'https://github.com/huynguyencong/DataCache.git', :tag => "#{s.version}" }
   s.author       = { 'Huy Nguyen Cong' => 'https://github.com/huynguyencong' }
+  s.osx.deployment_target = '10.11'
   s.ios.deployment_target = '13.0'
```

Comment:
is there a reason why swift package has ios9 but pod 13?
i don't know much about pod, so just wondered if i need to change osx target to match ios13 release date

Reply:
It should be the same deployment target between Podspec and SPM

Reply:
So you want me to change it back to ios9?

Reply:
I think iOS 13 is ok. I will update it later.